package.json

```diff
@@ -64,7 +64,7 @@
     "sinon": "^1.17.3"
   },
   "dependencies": {
-    "axe-core": "^2.5.0",
+    "axe-core": "^3.0.0-alpha.8",
```

Comment:
I think we should release this with 2.5.0. Maybe we could use `npm-install-version` to install `axe-core@next` so we can run that shadowDOM test separately. WDYT?

Reply:
Sounds like more complexity to me. But we can discuss it later.

Reply:
We talked about doing a release with axe-core 2.6 and the axe.run changes, and a separate Shadow DOM prerelease later. I've separated the changes into two PRs so we can do that.